packages/cli/tests/test.ts

```diff
@@ -21,74 +21,21 @@ describe('NgxsCli', () => {
   });
 
   it('Check correctly generated template app.actions.ts', () => {
-    const filePath = path.resolve(generatedFiles.actions);
-    const actionsOutput = fs.readFileSync(filePath).toString();
-
-    expect(
-      'export class AppAction {\n' +
-      "  static readonly type = '[App] Add item';\n" +
-      '  constructor(public payload: string) { }\n' +
-      '}\n'
-    ).to.equal(actionsOutput);
+    const actionAppSnapshot = fs.readFileSync(path.resolve(expectedFiles.actions)).toString();
+    const actionsAppOutputCli = fs.readFileSync(path.resolve(generatedFiles.actions)).toString();
+    expect(actionAppSnapshot).to.equal(actionsAppOutputCli);
```

Comment:
I like this, makes the tests much cleaner!

Reply:
done